2/2_create_bootstrap.md

```diff
@@ -1,13 +1,29 @@
-
-* What is bootstrap? why do we need it?
-* What properties does a boot startap node has
-* By default how libp2p already gives an ipfs address and how we can fix a address for the bootstrap for easier purpose
-* Make sure you have only one bootstrap running, because you cant have 2 nodes listening to the same port and address
+Bootstrap - what is it good for?
+================================
+
+If you've never heard the term 'bootstrap' before, don't worry.  While it can mean a variety of [different things](https://techterms.com/definition/bootstrap#:~:text=Terms%20%3A%20Bootstrap%20Definition-,Bootstrap,automatically%20loads%20and%20executes%20commands.), for purposes of networking and libp2p, think of a bootstrap node as an anchor point that guarantees your individual libp2p node will be able to connect to the broader network.  The basic idea of a bootstrap node is just a libp2p node with a publicly known address that any other node can use as a first point of contact to connect to the network.  In theory, new nodes can always count on the bootstrap node to be up and running when they want to connect to the network so you can hardcode a bootstrap node's multiaddress in your new node's starting configuration and be confident you will always have a connection point to the broader network.  
+
+By default, libp2p nodes generate new `PeerIds` and addresses every time they start unless you provide those details in the `options` object.  You can define them as below:
+```
+const node = libp2p.create({
+    addresses: {
+        listen: ['/ip4/127.0.0.1/tcp/63685', '/ip4/127.0.0.1/tcp/63786/ws]
+    },
+    peerId: {
+        "id": "Qma3GsJmB47xYuyahPZPSadh1avvxfyYQwk8R3UnFrQ6aP",
+        "privKey": "CAASpwkwgg...",
+        "pubKey": "CAASpgIwggEiMA0G..."
+    }   
+})
+```
```

Comment:
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting,  same as above

Reply:
Fixed.